scripts/travis_install.sh

```diff
@@ -6,7 +6,8 @@ function install_solc {
 }
 
 install_solc
+: ${PIP:="pip3"}
 
-pip install -U pip
-pip uninstall -y Manticore || echo "Manticore not cached"  # uninstall any old, cached Manticore
-pip install --no-binary keystone-engine -e .[dev]  # ks can have pip install issues
+$PIP install -U pip
```

Comment:
We need to install `z3-solver` here. The py3 job is failing at the moment because it's not available in the cache.